webapp/src/utils/api.ts

```diff
@@ -88,7 +89,15 @@ export const fetchApi =
     // fetch() might throw if the user is offline, or some unlikely networking error occurs, such a DNS lookup failure.
     // Let's also throw if the status is not OK, so it's uniform.
     if (!response.ok) {
-      throw Error(`${response.status} : ${response.statusText}`);
+      try {
+        const { detail: errorMessage } =
+          (await response.json()) as HTTPExceptionModel;
+        throw Error(errorMessage);
+      } catch {
+        throw Error(
+          `Failed to load resource: the server responded with a status of ${response.status} + ${response.statusText}. `
+        );
```

Comment:
I think this is quite verbose, and the "+" is particularly weird to me.
![image](https://user-images.githubusercontent.com/8386369/231320209-7f9e02aa-008a-45ba-9008-f311c73c3e37.png)
What do you think of something that simple?
```suggestion
        throw Error(`${response.status} ${response.statusText}`);
```
![image](https://user-images.githubusercontent.com/8386369/231321248-cf29ca1a-447a-497c-a8b3-aa686df9a02b.png)

Reply:
Okay, I can modify it as you suggest.